### DISCUSSION TITLE
about  the measurements, in the manual (end of page 6)

### DISCUSSION THREAD

**Discussion by ghost** [OpenHantek/OpenHantek6022#301] [Q&A] [answered]
I am learning and I would appreciate an explanation about  the measurements, in the manual (end of page 6) where it says "
The bottom line shows settings and results of measurement and calculation."...  thanks a lot

**Comment by Ho-Ro** [comment, marked as answer]
The meaning of each value is explained at the bottom of page 7. 
If you want to learn more you could read about the [mathematics of AC voltages](https://en.wikipedia.org/wiki/Alternating_current#Mathematics_of_AC_voltages) in Wikipedia and about [THD](https://www.ap.com/blog/thd-and-thdn-similar-but-not-the-same) (Total Harmonic Distortion) and see [calculation examplex](https://www.allaboutcircuits.com/technical-articles/the-importance-of-total-harmonic-distortion) for a square wave and a distorted sine signal.
If you want to see the *magic* behind just have a look at [the calculation](https://github.com/OpenHantek/OpenHantek6022/blob/main/openhantek/src/post/spectrumgenerator.cpp), I spread a lot of comments to guide the reader through the data flow.
